crowdaction/main_test.go

```diff
@@ -0,0 +1,294 @@
+// Code generated by private/model/cli/gen-api/main.go. DO NOT EDIT.
```

Comment:
How has this file been generated?
Are there any unit tests that use it or how is it used?

Reply:
no unit tests yet, don't know how to do them yet or whether in fact it's worth doing them now.
just wanted to have it there as TO DO and as a model

Reply:
Ok.  
Let's use `//TODO` (no space), so we find all of them with one search over the entire repo.